tutorials/pages/send-shimmer-tokens-with-iota-sdk.md

```diff
@@ -0,0 +1,148 @@
+```
+
+## Create an account
+
+Now that we have all details, let's go through the example code.
+
+After including the needed dependencies, we have have the main function that loads the environment variables from the `.env` file we created earlier. It uses the information to set up [Stronghold](/stronghold.rs/welcome) to store our seed safely.
+
+```js
+import { Wallet, CoinType, initLogger, WalletOptions } from '@iota/sdk';
+
+// This example uses secrets in environment variables for simplicity which should not be done in production.
+require('dotenv').config({ path: '.env' });
+
+const walletOptions: WalletOptions = {
+  storagePath: process.env.WALLET_DB_PATH,
+  clientOptions: {
+    nodes: [process.env.NODE_URL],
+  },
+  coinType: CoinType.Shimmer,
+  secretManager: {
+    stronghold: {
+      snapshotPath: process.env.STRONGHOLD_SNAPSHOT_PATH,
+      password: process.env.STRONGHOLD_PASSWORD,
+    },
+  },
+};
+
+const wallet = new Wallet(walletOptions);
+
+// A mnemonic can be generated with `Utils.generateMnemonic()`.
+// Store the mnemonic in the Stronghold snapshot, this needs to be done only the first time.
+// The mnemonic can't be retrieved from the Stronghold file, so make a backup in a secure place!
+await wallet.storeMnemonic(process.env.MNEMONIC);
+
+// Create a new account
+const account = await wallet.createAccount({
+  alias: 'Alice',
+});
+console.log('Generated new account:', account.getMetadata().alias);
+```
+
+If everything worked correctly, you will see the message `Generated a new account` and you will find a Stronghold file and a database directory have been created to store the current state of your wallet.
+
+## Generate an address
+
+In this step, we will generate a new address to receive some testnet tokens. For this we will use the second example called `02_generate_address.rs`.
+
+Here again we read the environment variables from the `.env` file and then we recreate the account manager which will use the Stronghold file and database that were created in the previous step.
+
+```js
+// To create an address we need to unlock stronghold.
+await wallet.setStrongholdPassword(process.env.STRONGHOLD_PASSWORD);
+
+const address = (await account.generateEd25519Addresses(1))[0];
+
+console.log(`Generated address:`, address.address);
+```
+
+You can see all testnet addresses begin with `rms`, which is the reverse of what real Shimmer addresses start with. This is how you can tell testnet and real addresses apart. You can get some testnet tokens from the [faucet](https://faucet.testnet.shimmer.network).
+
+## Check the balance
+
+Now you should have some tokens. To validate that, we can use the library to inspect our account.
+
+```js
+// Sync new outputs from the node.
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+const _syncBalance = await account.sync();
+
+// After syncing the balance can also be computed with the local data
+const balance = await account.getBalance();
```

Comment:
Should we explain that calling either of them would be enough?

Reply:
Yes.